ocs_ci/ocs/resources/pod.py

```diff
@@ -1360,3 +1360,23 @@ def delete_deploymentconfig_pods(pod_obj):
                 dc_ocp_obj.wait_for_delete(
                     resource_name=pod_obj.get_labels().get('name')
                 )
+
+
+def wait_for_new_osd_pods_to_come_up(number_of_osd_pods_before):
+    status_options = ['Init:1/4', 'Init:2/4', 'Init:3/4', 'PodInitializing', 'Running']
+    try:
+        for osd_pods in TimeoutSampler(
+            timeout=180, sleep=3, func=get_osd_pods
```

Comment:
maybe sample more frequently for better chances not to miss 1 OSD coming up before all do? how about every 2 seconds instead of 3?

Reply:
I think 3 seconds is the minimum, cause it takes like 1-2 seconds to execute the 'oc' command